app/src/pages/See_User_Reviews.py

```diff
@@ -0,0 +1,98 @@
+import streamlit as st
+import requests
+from requests.exceptions import ConnectionError
+from modules.nav import SideBarLinks
+
+SideBarLinks()
+
+def get_reviewer_id_by_name(author_name):
+    try:
+        response = requests.get(f'http://api:4000/rver/reviewers?name={author_name}')
+        response.raise_for_status()
+        author_data = response.json()
+        if author_data:
+            return author_data[0]['reviewerID']
+        else:
+            st.error("Reviewer not found")
+            return None
+    except requests.exceptions.RequestException as e:
+        st.error(f"Failed to fetch reviewer data: {e}")
+        return None
+
+def get_reviews_by_reviewer_id(reviewer_id):
+    try:
+        response = requests.get(f'http://api:4000/r/reviews?authorID={reviewer_id}')
+        response.raise_for_status()
+        return response.json()
+    except requests.exceptions.RequestException as e:
+        st.error(f"Failed to fetch reviews: {e}")
+        return []
+
+def delete_review(review_id):
+    try:
+        response = requests.delete(f'http://api:4000/r/reviews/{review_id}')
+        response.raise_for_status()
+        st.success("Review deleted successfully!")
+    except requests.exceptions.RequestException as e:
+        st.error(f"Failed to delete review: {e}")
+
+def update_review(review_id, review_data):
+    try:
+        response = requests.put(f'http://api:4000/r/reviews/{review_id}', json=review_data)
+        response.raise_for_status()
+        st.success("Review updated successfully!")
+        st.experimental_rerun()
+    except requests.exceptions.RequestException as e:
+        st.error(f"Failed to update review: {e}")
+
+def main():
+    st.title("Your Reviews")
+    
+    author_name = "Ally Descoteaux"
+    reviews = get_reviews_by_reviewer_id(1)
+        
+    if reviews:
+            for review in reviews:
+                st.subheader(f"Review ID: {review['reviewID']}")
+                st.write(f"Title: {review['title']}")
+                st.write(f"Rating: {review['rating']}")
+                st.write(f"Recommend: {review['recommend']}")
+                st.write(f"Pay Type: {review['pay_type']}")
+                st.write(f"Pay: {review['pay']}")
+                st.write(f"Job Type: {review['job_type']}")
+                st.write(f"Date: {review['date_time']}")
+                st.write(f"Verified: {review['verified']}")
```

Comment:
I'm pretty sure a reviewer shouldn't be able to verify themselves. I feel like it makes more sense if all reviews should default to False until an admin approves.